package.json

```diff
@@ -59,6 +59,10 @@
     "@folio/eslint-config-stripes": "^3.2.1",
     "@folio/stripes-cli": ">=1.5.0",
     "eslint": "^5.6.1",
+    "react": "~16.6.0",
+    "react-dom": "~16.6.0", 
+    "react-redux": "~5.1.1", 
+    "redux": "~3.7.2",
```

Comment:
@OleksandrAntonenko1 were these `devDependencies` additions intentional? I'm now getting messages like:
```
warning @folio/platform-core@1.1.2-SNAPSHOT: "dependencies" has dependency "react" with range "^16.6.3" that collides with a dependency in "devDependencies" of the same name with version "~16.6.0"
```
They're already defined in `devDependencies`.